blueocean-core-js/src/js/components/ContentPageHeader.jsx

```diff
@@ -26,7 +27,9 @@ export const SiteHeader = props => {
     return (
         <BasicHeader className="ContentPageHeader">
             <TopNav>
-                <BlueLogo href={props.homeURL} />
+                <Extensions.Renderer extensionPoint="jenkins.header.logo">
+                    <BlueLogo href={props.homeURL} />
```

Comment:
Yes. Initially I put that functionality in for an editor demo, but I definitely had logo-replacement in mind, I'm glad it's going in.